os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -82,15 +82,90 @@ static trwifi_scan_list_s *g_scan_list;
 static int g_scan_num;
 extern struct netdev *ameba_nm_dev_wlan0;
 
+#include "freertos/wrapper.h"

```

Comment:
how about to move include to line 28. I think gathering header will be better to maintain code.

Reply:
Done.